README.md

```diff
@@ -9,63 +9,93 @@
-spring.datasource.username=postgres
-spring.datasource.password=1234 
+docker-compose up -d
 ```
 
-There is also the need to update `ion.core-base-url` and `ion.core-token`. Check [I-On Core documentation](https://github.com/i-on-project/core/blob/master/README.md)
+The **first time you build the images may take a while** but you'll notice following runs are much faster due to the way Docker caches its image layers.
 
-Finally `spring.mail.username`, `spring.mail.password` and `email.sender` for the email user that will send the alert emails.
+#### Customizing the containers
+While the environment works out of the box for most setups you may run into **port binding issues** if you're already using any of the ports we configured **Docker** to use (**8080** for the local git server and **5432** for the Postgres database).
 
-## Build
+These ports are configured from **Docker environment variables** and their preset values can be viewed by running the `docker-compose config` command or by reading the [.env](.env) file (which is a simple list of key-value pairs). Simply edit the **.env** file with the port values you wish to use and you're good to go.
+
+**Note**: All port binding commands in docker use the `HOST:CONTAINER` format so, for example, the binding `1337:80` would bind the **1337 port** on the **host** machine to the **80 port** inside the **container**.
+
+The default database, username and password are also customizable through the **.env** file.
+
+#### Accessing the containerized database
+While the Integration app will be able to connect to the database out of the box without any human intervention you might also be interested in connecting directly to run your own queries.
+
+You can use your favorite **SQL client** to connect to the containerized database just as you would for any other database server.
+
+The **default** JDBC connection string is:
+```
+jdbc:postgresql://ion-db:5432/ion_integration
+```
 
-    $ git clone git@github.com:i-on-project/integration.git
-    $ cd integration
-    
-You can build via command line
+Or, if you're using a client that supports PostgreSQL Drivers (like [DBeaver](https://dbeaver.io/)) you can simply connect to a PostgreSQL destination with the following parameters:
+- **Host**: localhost
+- **Port**: (SQL_PORT in .env file)
+- **Database**: (SQL_DATABASE in .env file)
+- **Authentication**: Database Native
+- **Username**: (SQL_USER in .env file)
+- **Password**: (SQL_PASSWORD in .env file)
 
-    $ ./gradlew build
-    
-Or using an IDE with gradle support such as [IntelliJ IDEA](https://www.jetbrains.com/idea/) or [Eclipse](https://www.eclipse.org/ide/)
+### Stopping and removing the environment
+If you are running in **attached** mode (without the `-d` flag) you can use `Ctrl + C` to stop all containers **gracefully** or, if you're in a hurry (or a container is not stopping on its own) you can press `Ctrl + C` to **force shutdown**.
```

Comment:
If you are running in **attached** mode (without the `-d` flag) you can use `Ctrl + C` to stop all containers **gracefully** or, if you're in a hurry (or a container is not stopping on its own) you can press `Ctrl + C`  a second time to **force shutdown**.

Reply:
Fixed